docs/docs/porting-to-via.md

```diff
@@ -176,6 +177,7 @@ Start with the following JSON template:
     "layouts": {
         "labels":
         "keymap":
+        [ Paste your data here!]
```

Comment:
I'm not sure I understand this change, after you download JSON from kle it already has the correct number of brackets, I feel like adding [ paste your data ] would just invite more confusion and ppl adding an extra set of brackets.

Reply:
Well, people keep pasting in the wrong place, so... But agreed the brackets shouldn't be there...